app/src/main/java/com/x8bit/bitwarden/data/platform/manager/BiometricsEncryptionManagerImpl.kt

```diff
@@ -58,6 +59,10 @@ class BiometricsEncryptionManagerImpl(
             // user removed all biometrics from the device
             settingsDiskSource.systemBiometricIntegritySource = null
             return null
+        } catch (e: ProviderException) {
+            // key generation failed because user cannot be authenticated by the OS.
+            settingsDiskSource.systemBiometricIntegritySource = null
+            return null
```

Comment:
🗒️ Unable to reproduce the reported crash. However, handling the exception should prevent it from occurring if the scenario arises again in production. Should it manifest, the app will exhibit the same behavior as the preceding `catch` block.

Reply:
Do we also want to catch generic errors?

Reply:
@shannon-livefront Ideally we'd only catch the specific errors we know can occur so that we don't mask new and different problems.